tests/scripts/beneficiary/remote_import_test.py

```diff
@@ -1098,7 +1106,9 @@ def test_beneficiary_is_created_if_duplicates_are_found_but_id_is_in_retry_list(
         users_factories.UserFactory(
             email="unexistant@example.com", dateOfBirth=eighteen_years_in_the_past, firstName="Jane", lastName="Doe"
         )
-        users_factories.UserFactory(firstName="Jane", lastName="Doe", email="jane.doe@example.com")
+        users_factories.UserFactory(
+            firstName="Jane", lastName="Doe", dateOfBirth=eighteen_years_in_the_past, email="jane.doe@example.com"
```

Comment:
```suggestion
            firstName="Jane", lastName="Doe", dateOfBirth=AGE18_ELIGIBLE_BIRTH_DATE, email="jane.doe@example.com"
```
et supprimer `eighteen_years_in_the_past`